app/frontend/components/pages/sendAda/confirmTransactionDialog.tsx

```diff
@@ -84,21 +92,14 @@ class ConfirmTransactionDialogClass extends Component<Props, {}> {
             </Fragment>
           )}
 
-          {txConfirmType === 'send' && (
-            <Fragment>
-              <div className="ada-label">Amount</div>
-              <div className="review-amount">{printAda(summary.amount)}</div>
-            </Fragment>
-          )}
-
           {txConfirmType === 'convert' && (
```

Comment:
seems like after removing the conditional component above the two fragments can be merged together as both are conditioned by `txConfirmType` being "convert"

Reply:
I think there is a comment that this should be refactored so didn't want to make it an issue of this PR. Anyway, fixed and we ll refactor it in the future. Thnx.